app/presenters/two_factor_authentication/sms_selection_presenter.rb

```diff
@@ -4,6 +4,15 @@ def method
       :sms
     end
 
+    def info
+      if configuration.present?
+        t(
+          'two_factor_authentication.login_options.sms_info_html',
+          phone: configuration.masked_phone,
+        )
+      end
```

Comment:
_Probably_ not necessary given how we're currently using these presenters only when a configuration is present, but if the parent class's `info` is meant to handle the absence of any configuration, we could defer to `super` in an `else`? I don't feel strongly.

```suggestion
      else
        super
      end
```

(Same applies for the other presenter)